lib/determinator/retrieve/dynaconf.rb

```diff
@@ -0,0 +1,31 @@
+require 'faraday'
+
+module Determinator
+  module Retrieve
+    # A class which loads features from Dynaconf server
+    class Dynaconf
+      # @param :host [String] The host for local Dynaconf server
+      # @param :client [String] Faraday client instance, defaults to a new instance
+      def initialize(host:, client: default_client)
+        @host = host
+        @client = client
+      end
+
+      def retrieve(feature_id)
+        url = "http://#{@host}/scopes/florence-#{feature_id}/feature"
```

Comment:
This feels like we should have the protocol configurable as well (`http` or `https`)

Reply:
I think also in relation with @tompave-roo 's comment bellow, I'll rename the constructor parameter to `base_url` and document that it requires protocol, host and port, essentially everything before `/scopes` so that it provides the most flexibility.

Reply:
Suggestion: how do you feel about making this a bit stricter on what kind of feature IDs it accepts?

```ruby
feature = "aa%*&ciao"
URI.join("https://#{@host_port}", "/scopes/forence-#{feature}/", "feature")
# URI::InvalidURIError: bad URI(is not URI?): /scopes/forence-aa%*&ciao/
```

Reply:
`feature` is not a user input, it's a developer input and I see more disadvantages on making assumptions here about what Florence allows. If you provide a completely invalid name, you'll discover it during your development cycle, not in production.

Reply:
The extra check will make it slower in `production`, that's what I was trying to say ☝️ :-)